--- vboot-utils-a1c5f7c/Makefile.orig	2017-06-04 14:06:10.707762796 +0100
+++ vboot-utils-a1c5f7c/Makefile	2017-06-04 14:07:04.347718848 +0100
@@ -227,12 +227,6 @@
 CXX ?= g++
 PKG_CONFIG ?= pkg-config
 
-# Static?
-ifneq (${STATIC},)
-LDFLAGS += -static
-PKG_CONFIG += --static
-endif
-
 # Determine QEMU architecture needed, if any
 ifeq (${ARCH},${HOST_ARCH})
   # Same architecture; no need for QEMU
@@ -999,7 +993,6 @@
 # Link tests for external repos
 ${BUILD}/host/linktest/extern: ${HOSTLIB}
 ${BUILD}/host/linktest/extern: LIBS = ${HOSTLIB}
-${BUILD}/host/linktest/extern: LDLIBS += -static
 TEST_OBJS += ${BUILD}/host/linktest/extern.o
 
 .PHONY: hostlib
@@ -1090,9 +1083,6 @@
 ${UTIL_BINS} ${UTIL_BINS_STATIC}: ${UTILLIB}
 ${UTIL_BINS} ${UTIL_BINS_STATIC}: LIBS = ${UTILLIB}
 
-# Utilities for auto-update toolkits must be statically linked.
-${UTIL_BINS_STATIC}: LDFLAGS += -static
-
 
 .PHONY: utils
 utils: ${UTIL_BINS} ${UTIL_SCRIPTS}
@@ -1124,7 +1114,7 @@
 
 ${FUTIL_STATIC_BIN}: ${FUTIL_STATIC_OBJS} ${UTILLIB}
 	@${PRINTF} "    LD            $(subst ${BUILD}/,,$@)\n"
-	${Q}${LD} -o $@ ${CFLAGS} ${LDFLAGS} -static $^ ${LDLIBS}
+	${Q}${LD} -o $@ ${CFLAGS} ${LDFLAGS} $^ ${LDLIBS}
 
 ${FUTIL_BIN}: LDLIBS += ${CRYPTO_LIBS} ${FWLIB20}
 ${FUTIL_BIN}: ${FUTIL_OBJS} ${UTILLIB} ${FWLIB20} ${UTILBDB}
