--- vboot-utils-a1c5f7c/Makefile.orig	2017-06-04 13:24:39.666009720 +0100
+++ vboot-utils-a1c5f7c/Makefile	2017-06-04 13:25:23.960977143 +0100
@@ -158,7 +158,7 @@
 else
 # FIRMWARE_ARCH not defined; assuming local compile.
 CC ?= gcc
-CFLAGS += -DCHROMEOS_ENVIRONMENT -Wall ${WERROR} ${DEBUG_FLAGS}
+CFLAGS += $(COMMON_FLAGS) -DCHROMEOS_ENVIRONMENT -Wall ${WERROR} ${DEBUG_FLAGS}
 endif
 
 ifneq (${CUSTOM_MUSIC},)
